DEVNOTES.md

```diff
@@ -348,7 +348,7 @@ the docker image using `cli` parameter and connect to nodes using their names fr
 docker compose -f packaging/docker/docker-compose.yml up -d
 docker run -it --rm --net ignite3_default apacheignite/ignite3 cli
 > connect http://node1:10300
-> cluster init --name cluster --metastorage-group=node1,node2,node3
+> cluster init --name cluster
```

Comment:
Just thinking: What if the `name` is also optional? If I don't care about name - it might be autogenerated. Like docker does for containers.

Reply:
I thought about this but decided not to do it in the same patch. Agree that Docker's approach is pretty good, we use it in Ignite 2 for cluster tags (basically cluster name) as well. On the other hand, it's an easy concept for the user to understand.